### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by VishnuVandith** [qdrant/qdrant#1919] [General]
Hi all I'm a full stack developer. With 2 years if experience I want to contribute to Qdrant. Pls let me know how to get started

**Comment by rainleander** [comment]
Hey @VishnuVandith thanks for asking! 

If you are a Qdrant user - Data Scientist, ML Engineer, or MLOps, the best contribution would be the feedback on your experience with Qdrant. Let us know whenever you have a problem, face an unexpected behavior, or see a lack of documentation. You can do it in any convenient way - create an [issue](https://github.com/qdrant/qdrant/issues), start a [discussion](https://github.com/qdrant/qdrant/discussions), or drop up a [message](https://discord.gg/tdtYvXjC4h). 

If you use Qdrant or Metric Learning in your projects, we'd love to hear your story! Feel free to share articles and demos in our community via [discord](https://discord.gg/tdtYvXjC4h).

Are you familiar with Rust? Check out the [contribution guide](https://github.com/qdrant/qdrant/blob/master/CONTRIBUTING.md). If you have problems with code or architecture understanding - reach us at any time via [discussion](https://github.com/qdrant/qdrant/discussions) or [discord](https://discord.gg/tdtYvXjC4h). 

Feeling confident and want to contribute more? Come [work with us](https://qdrant.join.com/)!
